toolkit/scripts/toolchain.mk

```diff
@@ -159,6 +159,8 @@ $(final_toolchain): $(raw_toolchain) $(BUILD_SRPMS_DIR)
 	$(if $(filter y,$(UPDATE_TOOLCHAIN_LIST)), ls -1 $(toolchain_build_dir)/built_rpms_all > $(MANIFESTS_DIR)/package/toolchain_$(build_arch).txt)
 	touch $@
 
+.SILENT: $(toolchain_rpms)
```

Comment:
This disables the verbose mode of the toolchain extraction. Makes the output much smaller and much easier to understand.

Reply:
Oh, that's handy. I'm stealing that for some of my logging changes.;)